guava/src/com/google/common/base/Preconditions.java

```diff
@@ -109,7 +113,6 @@
  * @author Kevin Bourrillion
  * @since 2.0
  */
-@AnnotatedFor({"nullness"})
```

Comment:
Why did you remove this annotation?

Reply:
I removed it for it was missing in the google/guava. I think it was wrong to do so, I should revert this